### PR TITLE
Include content id when getting content from the DV

### PR DESCRIPTION
**Pull request by javiesses**
- [x] Provider
- [x] Service
- [x] Client
- [x] READMEs

### CODE REVIEW

File: modules/ipfs-cpinner-service/src/api.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function setupPublicApi (app: Express, provider: IpfsPinnerProvider, logg
 
     const content = await provider.get(did, key)
 
-    res.status(200).json({ content })
+    res.status(200).json(content)
```
Comment (ilanolkies) [inline modules/ipfs-cpinner-service/src/api.ts:17]: Why?

File: modules/ipfs-cpinner-provider/src/IpfsPinnerProvider.ts [hunk 2]
```diff
@@ -20,10 +20,10 @@ export default class IpfsPinnerProvider {
     return cid
   }
 
-  async get (did: DID, key: Key): Promise<Content[]> {
+  async get (did: DID, key: Key): Promise<SavedContent[]> {
     const cids = await this.metadataManager.find(did, key)
 
-    return Promise.all(cids.map(cid => this.ipfsClient.get(cid))).then(arrays => Array.prototype.concat.apply([], arrays))
+    return Promise.all(cids.map(cid => this.ipfsClient.get(cid).then(([content]) => ({ id: cid, content }))))
```
Comment (ilanolkies) [inline modules/ipfs-cpinner-provider/src/IpfsPinnerProvider.ts:26]: Improved legibility:

```suggestion
    return Promise.all(
      cids.map(cid => this.ipfsClient.get(cid)
        .then(([content]) => ({ id: cid, content }))
      )
    )
```

**Review by ilanolkies — CHANGES_REQUESTED** [review]
Great!

File: modules/ipfs-cpinner-provider/src/IpfsPinnerProvider.ts (hunk 2 above)
Comment (ilanolkies) [reply modules/ipfs-cpinner-provider/src/IpfsPinnerProvider.ts:26]: I'm now reading `ipfsClient` implementation again and something seems weird. We are getting many files iterating but then using just the first one

File: modules/ipfs-cpinner-service/src/api.ts (hunk 1 above)
Comment (javiesses) [reply modules/ipfs-cpinner-service/src/api.ts:17]: Before:
```
content: [
  'this is a content',
  'this is another content'
]
```
By returning everything under `content` we are specifying that it is an array of contents

Now:
```
[
  { id: '123456789', content: 'this is a content' },
  { id: '987654321', content: 'this is another content' }
]
```
Now we are specifying the meaning inside the object

Comment (ilanolkies) [reply modules/ipfs-cpinner-service/src/api.ts:17]: Brutal change, ok! Let's document it wisely.

**Review by ilanolkies — APPROVED** [review]
Makes much more sense 😄 LGTM!

**Pull request merged**